.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+language: python
+python:
+  - "3.4"
```

Comment:
I think one of the used libraries doesn't work under higher python version but i can try that